internal/transformer/classes/external/starlark.go

```diff
@@ -291,7 +339,8 @@ func (t *Starlark) addAppModules() {
 	t.StarGlobals[types.AppNameShort] = &starlarkstruct.Module{
 		Name: types.AppNameShort,
 		Members: starlark.StringDict{
-			qaFnName: t.getStarlarkQuery(),
+			qaFnName:    t.getStarlarkQuery(),
+			writeFnName: t.getStarlarkFileWrite(),
```

Comment:
This function should come under fs module. Can we add it inside addFSModules function?